src/object_store/Cargo.toml

```diff
@@ -43,5 +43,5 @@ tracing = "0.1"
 # [package.metadata.cargo-udeps.ignore]
 # normal = ["workspace-hack"]
 
-[features]
-hdfs-backend = ["opendal/services-hdfs"]
+# [features]
+# hdfs-backend = ["opendal/services-hdfs"]
```

Comment:
Only these changes are needed. We don't need to comment out the coddes in other files.